backend/apps/common/services.py

```
@@ -130,9 +137,18 @@ def process_password_reset_request(email: str, request):
                 recipient_list=[user.email],
                 fail_silently=False,
             )
-            logger.info(f"Correo de recuperación enviado a: {user.email}")
+            return (
+                True,
+                "Si el email está registrado, recibirás un correo con las instrucciones.",
+            )
+
         except Exception as e:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)